docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding Icon | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `leading` parameter to an `Icon`.
+
+### Text attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `title` parameter to a `Text` widget.
+**Color**      | Follow the steps above and then within the Text widget fill out the `style` parameter.
+**Typography** | Follow the steps above and then within the Text widget fill out the `style` parameter.
+
+### Divider attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Add Divider | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuDivider`.
+
+### Command attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding Icon | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `PopupMenuItem`. Within the child parameter add a `ListTile` widget and set the `trailing` parameter to an `Icon`.
+
+### Cascading menu indicator attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding cascading menu | Not currently supported.
+
+### Selection state attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+ | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `CheckedPopupMenuItem`.
+
+## Types
+
+Menus allow users to make a selection from multiple options. They are less prominent and take up less space than selection controls, such as a set of radio buttons.
+
+There are two types of menus: 1\. [dropdown menus](#dropdown-menus), 2\. [exposed dropdown menus](#exposed-dropdown-menus).
+
+![Composite image of example dropdown and exposed dropdown menu types](assets/menus/menu_types_composite.png)
+
+### Dropdown menus
+
+Dropdown menus display a list of options, triggered by an icon, button, or action. Their placement varies based on the element that opens them.
+
+#### Dropdown menu example
+
+The following is an example of a dropdown menu:
+
+- [Class definition](https://api.flutter.dev/flutter/material/PopupMenuButton-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/popup_menu.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/menus/dropdown&gh_ref=develop)
+
+![Dropdown menu example](assets/menus/dropdown_menu_example.png)
```

Comment:
Looks like the appbar color in the background is not the correct color, i.e. the material purple color. Is it correct in the demo?

Reply:
Done